Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,29 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Tom Jefferson is 19 years old, his weight is 50 kg and his height is 156.5 cm. 
+            string Name1 = "Tom Jefferson";
+            int Age1 = 19;
```

Comment:
local variables should be named camelCase, not PascalCase.

Reply:
I'm having a problem where visual studio says : The application is in break mode
I've tried to fix it, but nothing worked

Reply:
Are you writing code while debugging? Maybe that's why?

Reply:
I don't know how but I fixed it somehow

Reply:
Well restart the VS or a PC next time if you have issues like that. Happens.

Reply:
I redid the code for everything